src/network/ssl/socket.c

```diff
@@ -362,7 +361,7 @@ verify_callback(int preverify_ok, X509_STORE_CTX *ctx)
 		sk_GENERAL_NAME_pop_free(alts, GENERAL_NAME_free);
 	}
 	if (!matched && !saw_dns_name) {
-
+		X509_NAME *name;
```

Comment:
This is where is was originally. I was trying to figure out what to do with the wincrypt.h collision and accidentaly moved it. Somehow I've deeply rooted the rule to put the variables on top of the function. ;-]